### DISCUSSION TITLE
How to hide the green lab flask icon of Thorium Experiments menu?

### DISCUSSION THREAD

**Discussion by Unknown78** [Alex313031/Thorium-Win#182] [Q&A] [answered]
![Screenshot 2024-03-06 135001](https://github.com/Alex313031/Thorium-Win/assets/25132599/76625d60-baf1-4c64-876e-1ad9c8cf7261)

Just a personal taste.
Where could I find the settings to hide and unhide it?

**Comment by megaelk** [comment, marked as answer]
@Unknown78 should be able to remove this by changing `chrome://flags/#chrome-labs` to Disabled

**Reply by Unknown78** [reply]
It's working, thank you.
